components/marketplace/list-dialog-settings-form.tsx

```diff
@@ -101,6 +100,14 @@ export default function ListDialogSettingsForm({
   };
 
   const validateUnitsMaxPerOrder = (): [boolean, React.ReactNode] => {
+    if (!_unitsMaxPerOrder) {
+      return [false, "Must be more than 0."];
+    }
+
+    if (!_unitsMinPerOrder) {
```

Comment:
nit: The function is called `validateUnitsMaxPerOrder()` but is now also checking `_unitsMinPerOrder`